components/Container/ContentContainer.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+
+import React from 'react';
+
+
+/**
+ * ContentContainer is the base component that is meant to hold content
+ * inside a LayoutContainer.
+ */
+interface ContentContainerProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
L10: @andrewebdev  Looks like this interface is used in LayoutWithPanel, so you should probably export from here and import it there.